lib/imageProcessor.js

```diff
@@ -166,14 +168,21 @@ ImageProcessor.prototype.handleAvatarUpload = function(file, destDir) {
 	})
 }
 
-ImageProcessor.prototype.createScaledAvatar = function(path) {
+ImageProcessor.prototype.handleUserAvatarUpload = function(file, destDir) {
+	return this._handleProfileImageUpload(file, destDir, 128, 128, this.defaultGravity)
+}
+ImageProcessor.prototype.handleUserHeaderUpload = function(file, destDir) {
+	return this._handleProfileImageUpload(file, destDir, 1440, 340, this.defaultGravity)
```

Comment:
don't need to pass in the defaults.. my whole idea was to get rid of this totally unused argument.